dpgen/simplify/simplify.py

```diff
@@ -317,7 +317,7 @@ def make_fp_labeled(iter_index, jdata):
     create_path(work_path)
     picked_data_path = os.path.join(iter_name, model_devi_name, picked_data_name)
     os.symlink(os.path.abspath(picked_data_path), os.path.abspath(
-        os.path.join(work_path, "task." + data_system_fmt % 0)))
+        os.path.join(work_path, "task." + fp_task_fmt % 0)))
```

Comment:
I don't get the point. If you wants `task.000.000000`, it should be `data_system_fmt % 0 + fp_task_fmt % 0`.

Reply:
sorry, it was meant to be  `fp_task_fmt % (0,0)`